### ISSUE TITLE
Website graphics rendering issues (Linux)

### ISSUE THREAD

**Issue by smith558** [pmndrs/zustand#2249]
![image](https://github.com/pmndrs/zustand/assets/44023416/9f672778-b9c4-4cdb-907d-05c54956412d)

**Comment by elamandeep** [comment]
@smith558 In Ubuntu, it's okay. Can u share more details like os, browser etc?
![image](https://github.com/pmndrs/zustand/assets/68345524/a43ff721-dd33-4919-acfe-8894458206d9)

**Comment by smith558** [comment]
I am actually on Ubuntu.
```
OS: Ubuntu 23.10 x86_64 
Host: HP EliteDesk 805 G6 Desktop Mini PC SBKPF 
Kernel: 6.5.0-14-generic 
Uptime: 16 hours, 42 mins 
Packages: 3070 (dpkg), 59 (flatpak), 35 (snap) 
Shell: bash 5.2.15 
Resolution: 1920x1080 
DE: GNOME 45.1 
WM: Mutter 
WM Theme: Adwaita 
Theme: Yaru-blue [GTK2/3] 
Icons: Yaru-blue [GTK2/3] 
Terminal: gnome-terminal 
CPU: AMD Ryzen 7 PRO 4750G with Radeon Graphics (16) @ 4.454GHz 
GPU: AMD ATI 06:00.0 Renoir 
Memory: 8144MiB / 15280MiB 
Browser: Chrome Version 120.0.6099.109 (Official Build) (64-bit)
```

**Comment by smith558** [comment]
Solved with Linux kernel `6.7`
